docs/upgrade/v1-2-2-to-v1-3-1.md

```diff
@@ -87,3 +87,9 @@ To resolve this issue, perform the following steps:
   - [[BUG] Upgrade v1.2.2->v1.3.1 stuck at Waiting for kubelet to update, even the node has been updated](https://github.com/harvester/harvester/issues/6041)
 
 ---
+
+### 2. Automatic image cleanup is not functioning
+
+Due to an incomplete image list packaged in the published ISO image, the automatic image cleanup procedure will not work when users upgrade Harvester from v1.2.2 to v1.3.1. This does not impact the Harvester upgrade as a whole. To clean up container images manually after the upgrade ends, please refer to the [script](https://github.com/harvester/upgrade-helpers/blob/main/bin/harv-purge-images.sh). For more information, see [issue #6620](https://github.com/harvester/harvester/issues/6620).
```

Comment:
```suggestion
Automatic image cleanup, which is part of the upgrade process, cannot be performed if an incomplete image list is packaged in the published Harvester ISO. This issue does not block the upgrade, and you can use a [script](https://github.com/harvester/upgrade-helpers/blob/main/bin/harv-purge-images.sh) to manually clean up container images after the upgrade is completed. For more information, see [issue #6620](https://github.com/harvester/harvester/issues/6620).
```

Reply:
Sorry, I cannot tell the vibe of the suggested version here. I'll make my point: it's our oversight not to pack the image list in the ISO image. It's not the user's misoperation that causes the issue, and they're not expected to do anything to fix it. Also, only the affected upgrade paths will have this known issue updated in their respective documentation pages, so I think there's no need to generalize it by adding an "if." Please ignore the comment if I totally get your point wrong. Thank you.

Reply:
I was not going for a specific "vibe". I simply misunderstood what supposedly happens. Here's the new suggestion:

Because the published Harvester ISO contains an incomplete image list, automatic image cleanup cannot be performed during an upgrade from v1.2.2 to v1.3.1. This issue does not block the upgrade, and you can use [this script](https://github.com/harvester/upgrade-helpers/blob/main/bin/harv-purge-images.sh) to manually clean up container images after the upgrade is completed. For more information, see [issue #6620](https://github.com/harvester/harvester/issues/6620).

Reply:
I updated the paragraph according to the suggestion. Thank you.